ColorPicker/ColorPicker/ColorPicker.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  ColorPicker.swift
+//  ColorPicker
+//
+//  Created by Daria Tokareva on 22.06.2021.
+//
+
+import UIKit
+
+protocol ColorPickerDelegate {
+    func colorPicker(colorPicker: ColorPicker, hueValue: CGFloat)
+}
```

Comment:
Для того чтобы создать слабую ссылку на делегат, тип, который реализует протокол ColorPickerDelegate должен быть классом (ограничить протокол с помощью AnyObject или UIViewController).
```suggestion
protocol ColorPickerDelegate: UIViewController {
    func colorPicker(colorPicker: ColorPicker, hueValue: CGFloat)
}
```